cns/restserver/api_test.go

```diff
@@ -570,270 +646,234 @@ func getInterfaceForContainer(t *testing.T, name string) error {
 	return nil
 }
 
-func TestSetOrchestratorType(t *testing.T) {
-	fmt.Println("Test: TestSetOrchestratorType")
+// Decodes service's responses to test requests.
+func decodeResponse(w *httptest.ResponseRecorder, response interface{}) error {
+	if w.Code != http.StatusOK {
+		return fmt.Errorf("Request failed with HTTP error %d", w.Code)
+	}
 
-	setEnv(t)
+	if w.Result().Body == nil {
+		return fmt.Errorf("Response body is empty")
+	}
 
-	err := setOrchestratorType(t, cns.Kubernetes)
+	return json.NewDecoder(w.Body).Decode(&response)
+}
+
+func setEnv(t *testing.T) *httptest.ResponseRecorder {
+	envRequest := cns.SetEnvironmentRequest{Location: "Azure", NetworkType: "Underlay"}
+	envRequestJSON := new(bytes.Buffer)
+	json.NewEncoder(envRequestJSON).Encode(envRequest)
+
+	req, err := http.NewRequest(http.MethodPost, cns.V2Prefix+cns.SetEnvironmentPath, envRequestJSON)
 	if err != nil {
-		t.Errorf("setOrchestratorType failed Err:%+v", err)
 		t.Fatal(err)
 	}
+
+	w := httptest.NewRecorder()
+	mux.ServeHTTP(w, req)
+	return w
 }
 
-func TestCreateNetworkContainer(t *testing.T) {
-	// requires more than 30 seconds to run
-	fmt.Println("Test: TestCreateNetworkContainer")
+// IGNORE TEST AS IT IS FAILING. TODO:- Fix it https://msazure.visualstudio.com/One/_workitems/edit/7720083
+// // Tests CreateNetwork functionality.
+
+/*
+func TestCreateNetwork(t *testing.T) {
```

Comment:
Rather than comment out the code, you could add
```go
t.Skip()
```
which ignores the test

Reply:
I know, but that will still run the test right? Noone looks into the unit tests pipeline for what tests are actually running. Also no one would read an existing uncommented test with first list as t.Skip(). Dont want devs to assume we have any coverage for these APIs.

Reply:
It does run the test up until skip is called, and it shouldn't count for coverage, but I think you are right that this does bring more attention to the fact that these are broken and need to be fixed than just skipping them